analysis/analysis.py

```diff
@@ -441,56 +441,55 @@ def get_word_frequency_differences(student_data):
 
 def mean_view_time_comparison(student_data):
     """
```

Comment:
1) "mean_view_time_comparison" does not tell me that this function deals with sentiments.  Is there a better name for that?

2) The linter is saying that there are too-many local variables.  That generally means that the function is trying to do too many things at once.  Are there other functions that are sub-functions of this?  For instance, it seems like there is a function that given a list of words gives the largest (from zero) sentiment score in the list of words -- is this something that could be abstracted out?